servers/display_server.h

```diff
@@ -293,6 +293,7 @@ class DisplayServer : public Object {
 	virtual Rect2i get_display_safe_area() const { return screen_get_usable_rect(); }
 
 	enum {
+		SCREEN_AUTO = -5, // Remembers last screen position.
```

Comment:
This enum if for usage with `_get_screen_index()`. The new value makes no sense in this context.

Reply:
should I make a new enum that essentially copies these values?
I'm trying to follow the values from this, which seem to be the same
https://github.com/godotengine/godot/pull/97333/files#diff-985565fb1f983acb2d87dbe8931c929d0f46fdecb1d99a31e495b151c0f52186R429

Reply:
No idea tbh, you could use `-10` (to make sure there are no conflicts in the future, as it's a fake value anyway) and just hard-code it without any enum. I mean, it used `-2` previously (near the code you linked).